packages/charts/react-charting/src/components/DeclarativeChart/PlotlySchemaAdapter.ts

```diff
@@ -281,6 +281,9 @@ export const transformPlotlyJsonToVSBCProps = (
         const color = getColor(legend, colorMap, isDarkTheme);
         mapXToDataPoints[x].lineData!.push({
           legend,
+          lineOptions: ['dot', 'dash', 'longdash', 'dashdot', 'longdashdot'].includes(series.line?.dash as Dash)
+            ? { strokeDasharray: '5', strokeLinecap: 'butt', strokeWidth: '2', lineBorderWidth: '4' }
```

Comment:
lets use the same dash type as provided by plotly. 
dot = dot representation and so on
https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-dasharray